<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookWindow</class>
 <widget class="QDialog" name="BookWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnShowBooks">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>N채yt채 kirjat</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textBooks">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="textName">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="textAuthor">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="textIsbn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Lis채채</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
